crates/forge/src/main.rs

```diff
@@ -91,6 +91,10 @@ struct TestArgs {
     #[arg(long, conflicts_with = "only_ignored")]
     include_ignored: bool,
 
+    /// Display more detailed info about used resources
+    #[arg(long)]
+    detailed_resources: bool,
```

Comment:
We could add that configuration option to Scarb.toml, seems tedious to always write it by hand